keyboards/gray_studio/think65/solder/keymaps/yt/config.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2022 Yt Liu
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define RGBLIGHT_LAYERS
+#define DISABLE_LEADER
```

Comment:
This can be removed.  It doesn't appear to actually do anything.
```suggestion
```

Reply:
This should still be removed.  It's not used in the core, at all. And it defaults to off anyways.